libs/unity/library/Runtime/Scripts/Media/VideoRenderer.cs

```diff
@@ -112,41 +112,41 @@ private void Start()
 
         private void OnEnable()
         {
-            if (Source != null)
-            {
-                if (Source is IVideoSource videoSrc)
-                {
-                    videoSrc.GetVideoStreamStarted().AddListener(VideoStreamStarted);
-                    videoSrc.GetVideoStreamStopped().AddListener(VideoStreamStopped);
-
-                    // If registering while the audio source is already playing, invoke manually
-                    if (videoSrc.IsStreaming)
-                    {
-                        VideoStreamStarted(videoSrc);
-                    }
-                }
-                else
-                {
-                    throw new ArgumentException("Video source does not implement IVideoSource.");
-                }
-            }
+            //if (Source != null)
+            //{
+            //    if (Source is IVideoSource videoSrc)
+            //    {
+            //        videoSrc.GetVideoStreamStarted().AddListener(VideoStreamStarted);
+            //        videoSrc.GetVideoStreamStopped().AddListener(VideoStreamStopped);
+
+            //        // If registering while the audio source is already playing, invoke manually
+            //        if (videoSrc.IsStreaming)
+            //        {
+            //            VideoStreamStarted(videoSrc);
+            //        }
+            //    }
+            //    else
+            //    {
+            //        throw new ArgumentException("Video source does not implement IVideoSource.");
+            //    }
+            //}
         }
 
         private void OnDisable()
         {
-            var videoSrc = (IVideoSource)Source;
-            if (videoSrc != null) // depends in particular on Unity's component destruction order
-            {
-                videoSrc.GetVideoStreamStarted().RemoveListener(VideoStreamStarted);
-                videoSrc.GetVideoStreamStopped().RemoveListener(VideoStreamStopped);
-            }
+            //var videoSrc = (IVideoSource)Source;
+            //if (videoSrc != null) // depends in particular on Unity's component destruction order
+            //{
+            //    videoSrc.GetVideoStreamStarted().RemoveListener(VideoStreamStarted);
+            //    videoSrc.GetVideoStreamStopped().RemoveListener(VideoStreamStopped);
+            //}
         }
 
-        private void VideoStreamStarted(IVideoSource source)
+        public void VideoStreamStarted(IVideoSource source)
```

Comment:
`VideoStreamStarted` => `Start` or `StartRendering` or similar, since it's now a publicly-invoked method (active; as opposed to an internally-invoked callback (passive)).

Reply:
Fixed